### PR TITLE
Movies display

### PR DESCRIPTION
**Pull request by yvette809**
I have shown the movies on the start page. I filtered them based on released date

### CODE REVIEW

File: src/js/movies.js [hunk 1]
```diff
@@ -1,3 +1,87 @@
...
+        this.imageUrl = imageUrl;
+        this.videoUrl = videoUrl;
+        this.genres = genres;
+        this.ageRestriction = ageRestriction;
+        this.releaseDate = releaseDate;
+        this.screeningTimes = screeningTimes;
+        this.tags = tags;
+        this.rating = rating;     
+
+    }
```
Comment (solidtop) [inline src/js/movies.js:27]: A thought regarding this constructor. The object/class Movies is as far I can see not created anywhere in the file. I see you used "static" on the functions so you don't need to create objects before using them :) So I think there's no need for the constructor in this case! What do you think? :P

Comment (yvette809) [reply src/js/movies.js:27]: I only created the constructor because it will be used for other functions in the future

Comment (solidtop) [reply src/js/movies.js:27]: Oh nice I see!
An alternative solution for how many parameters the constructor takes is to only take a "movie" object as parameter. Then you only need to set: this.movie = movie; and then you can use all the variables with e.g: movie.title, movie.rating.

Overall it looks good and easy to read! :)

Comment (yvette809) [reply src/js/movies.js:27]: Ok thanks for the suggestion

File: src/js/movies.js [hunk 2]
```diff
@@ -1,3 +1,87 @@
...
+      if(new Date(movie.releaseDate ).getFullYear() === new Date().getFullYear()){
+                  // get the heading
+       let heading = document.querySelector('.heading-1')
+         heading.innerText = 'På Bion Just Nu' 
+         Movies.createMovie(movie)
+         
+                
+            }else{
+            let heading = document.querySelector('.heading-2')
+            heading.innerText = 'kommer snårt'
```
Comment (solidtop) [inline src/js/movies.js:41]: A suggestion of change to "Kommer Snart" so it's more consistent to "På Bio Just Nu" :)

File: src/js/movies.js [hunk 3]
```diff
@@ -1,3 +1,87 @@
...
+        this.releaseDate = releaseDate;
+        this.screeningTimes = screeningTimes;
+        this.tags = tags;
+        this.rating = rating;     
+
+    }
+
+
+     static displayMovies(){
+        movies.movies.forEach(movie =>{
```
Comment (solidtop) [inline src/js/movies.js:31]: Another suggestion to make this function a bit more secure. "Movies" comes from outside of the function scope which can lead to problems in the future, if for example the movie data comes from outside the scope of this file (maybe an API object). 
A solution to this is to have "movies" as a parameter on the function "displayMovies" instead and thus making a dependency injection :)

**Review by solidtop — APPROVED** [review]
Code looks good! Although using static methods removes the need to create instances of "Movies" which makes it's constructor unused. 

Great Job! 👍

File: src/js/movies.js (hunk 2 above)
Comment (yvette809) [reply src/js/movies.js:27]: Can you be more explicit? I am not sure I understand what you mean

Comment (solidtop) [reply src/js/movies.js:27]: I'm referring to the typo in the string from heading.innerText = 'kommer snårt'. :) 
And also the string above it ("På Bio Just Nu") has a capital letter on every word. Only a small detail.

**Comment by yvette809** [comment]
I have created a single movie page on  moviePage.js and movie.html.  When you click the image on the start page, it will take you to the movie.html page.

I used local storage to save the data on movies.js and retrieved the data on moviePage.js using localStorage.get('movie')

I did just the javascript. No css yet. Please comment if something needs to be improve on

**Review by holst9559 — APPROVED** [review]
Looks good

**Pull request merged**